doc/source/reference/code_support.rst

```diff
@@ -12,66 +12,78 @@ as inputs.  Below is a list showing what level of support is provided for
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| Chombo                |     Y      |     Y     |      Y     |   Y   |    Y     |    Y     |     Y      |   Full      |
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| Enzo                  |     Y      |     Y     |      Y     |   Y   |    Y     |    Y     |     Y      |   Full      |
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| Enzo-E                |     ?      |     ?     |      ?     |   ?   |    ?     |    ?     |     ?      |   ?         |
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| Exodus II             |     ?      |     ?     |      ?     |   ?   |    ?     |    ?     |     ?      |   ?         |
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| FITS                  |     Y      |    N/A    |      Y     |   Y   |    Y     |    Y     |     Y      |   Full      |
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| FLASH                 |     Y      |     Y     |      Y     |   Y   |    Y     |    Y     |     Y      |   Full      |
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| Gadget                |     Y      |     Y     |      Y     |   Y   |    Y     |    Y     |     Y      |   Full      |
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| GAMER                 |     Y      |     Y     |      Y     |   Y   |    Y     |    Y     |     Y      |   Full      |
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| Gasoline              |     Y      |     Y     |      Y     |   Y   |    Y     |    Y     |     Y      |   Full      |
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| Gizmo                 |     Y      |     Y     |      Y     |   Y   |    Y     |    Y     |     Y      |   Full      |
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| Grid Data Format (GDF)|     Y      |    N/A    |      Y     |   Y   |    Y     |    Y     |     Y      |   Full      |
++-----------------------+------------+-----------+------------+-------+----------+----------+------------+-------------+
+| IAMR                  |     ?      |     ?     |      ?     |   ?   |    ?     |    ?     |     ?      | ?           |
```

Comment:
Just highlighting that this is where help is needed. I don't personally know how to complete this line either.